sqlmesh/core/node.py

```diff
@@ -186,12 +186,14 @@ class _Node(PydanticModel):
     __inferred_interval_unit: t.Optional[IntervalUnit] = None
 
     @field_validator("name", mode="before")
-    @classmethod
-    def _name_validator(cls, v: t.Any) -> str:
-        return v.meta.get("sql") or v.sql() if isinstance(v, exp.Expression) else str(v)
+    def _name_validator(cls, v: t.Any) -> t.Optional[str]:
+        if v is None:
+            return None
+        if isinstance(v, exp.Expression):
+            return v.meta.get("sql") or v.sql()
+        return str(v)
 
     @field_validator("start", mode="before")
-    @classmethod
```

Comment:
Why remove this?

Reply:
This is a best practice as suggested in pydantics docs.

Reply:
What is strange is that I don't even remember removing this so I can't explain why it was. Will revert.